api/notd/store/saver.py

```diff
@@ -437,3 +439,38 @@ async def create_user_interaction(self, date: datetime.datetime, userAddress: st
             signature=signature,
             message=message,
         )
+
+    async def create_latest_update(self, date: datetime.datetime, key: str, name: Optional[str], connection: Optional[DatabaseConnection] = None) -> LatestUpdate:
+        createdDate = date_util.datetime_from_now()
+        updatedDate = createdDate
+        values = {
+            LatestUpdatesTable.c.createdDate.key: createdDate,
+            LatestUpdatesTable.c.updatedDate.key: updatedDate,
+            LatestUpdatesTable.c.date.key: date,
+            LatestUpdatesTable.c.key.key: key,
+            LatestUpdatesTable.c.name.key: name,
+        }
+        query = LatestUpdatesTable.insert().values(values)
+        result = await self._execute(query=query, connection=connection)
+        latestUpdateId = result.inserted_primary_key[0]
+        return LatestUpdate(
+            latestUpdateId=latestUpdateId,
+            createdDate=createdDate,
+            updatedDate=updatedDate,
+            key=key,
+            name=name,
+            date=date,
+        )
+
+    async def update_latest_update(self, latestUpdateId: int, key: Optional[str] = None, name: Optional[str] = None, date: Optional[datetime.datetime] = None, connection: Optional[DatabaseConnection] = None) -> None:
```

Comment:
name needs to use the special thing we use like in metadata for nullable strings